plugins/org.eclipse.reddeer.core/src/org/eclipse/reddeer/core/condition/ActiveShellExists.java

```diff
@@ -31,7 +31,7 @@
 	@Override
 	public boolean test() {
 		shell = ShellLookup.getInstance().getCurrentActiveShell();
-		return shell != null;
+		return shell != null ? true : false;
```

Comment:
I thought so. I call test method in assert and get AssertionError: null. I have on idea what is going on here. In any case I am expecting boolean but gets null instead sometimes...